src/v0/gateway.ts

```diff
@@ -68,9 +100,12 @@ export function createGateway({
         await wait(attempts ** 2 * 1000);
         return post(path, body, attempts + 1, init);
       }
-      return Promise.reject(
-        new ManifoldError({ type: ErrorType.ServerError, message: resp.statusText })
-      );
+
+      throw new ManifoldError({ type: ErrorType.ServerError, message: resp.statusText });
+    }
+
+    if (resp.status < 200 || resp.status > 299) {
+      throw new RestError(resp.status, resp.statusText);
```

Comment:
not allowed any redirects? don't think they would happen but what if they do?

Reply:
I can't say I've ever encountered a redirect response for an AJAX API. I'm comfortable ignoring it.